### DISCUSSION TITLE
Pipeline execution and crash

### DISCUSSION THREAD

**Discussion by jp136769** [r-lidar/lasR#92] [Q&A]
Hello,

Is there any way to skip and continue when a pipeline fails on a file (for any reason, we can imagine a faulty bouding box for a point cloud file that makes a rasterization process fail).
Something similar to ``opt_stop_early(ctg)`` in ``lidR`` ?

Thanks

**Comment by Jean-Romain** [comment]
You can remove the file from the `LAScatalog`

**Reply by jp136769** [reply]
Let's say we have this simple script :

```r
INPUT_FOLDER <- utils::choose.dir(caption = "Select LAZ folder")
f <- list.files(INPUT_FOLDER, pattern = "*.laz", full.names = TRUE)
OUTPUT_FOLDER <- utils::choose.dir(caption = "Select output folder")
dir.create(OUTPUT_FOLDER)
ofile = paste0(OUTPUT_FOLDER, "/*_raster.tif")

pipeline <- lasR::rasterize(5, operators = "z_max", ofile = ofile)

# Set computing strategy globally
lasR::set_parallel_strategy(lasR::concurrent_files(8))

ans <- lasR::exec(pipeline, on = f, with = list(progress = TRUE))
```
I don't know, at execution time, wich file to remove from the list.

**Reply by Jean-Romain** [reply]
My bad I believed it was a lidR question. 

Well, please report the issue with a reproducible example. If R crashes it must be fixed not skipped.
